contrib/kind.yaml.j2

```diff
@@ -15,6 +15,9 @@ kubeadmConfigPatches:
   kind: ClusterConfiguration
   metadata:
     name: config
+  apiServer:
+    extraArgs:
+      "enable-admission-plugins": "ValidatingAdmissionWebhook"
```

Comment:
should we make this an option instead of deploying it by default?

Reply:
Personally, I am for deploying it with KIND so that we can validate it during our CI runs.

Reply:
Excuse me, I am also for having less configuration parameters in this project. So thus: just deploying it all the time.

Reply:
fair enough :+1: